.github/workflows/ci.yml

```
@@ -0,0 +1,40 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+*
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
+
+    steps:
+      - name: Install dependencies
+        run: sudo apt -y -qq install lintian
```

I prefer shell commands to be as verbose as possible for readability purposes.

```suggestion
        run: sudo apt-get --yes --quiet install lintian
```

Also suggesting `apt-get` instead of `apt` since `apt` likes to emit a warning about using it in shell scripts.